src/Integration/LanguageConverter.cs

```diff
@@ -0,0 +1,68 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="LanguageConverter.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+
+namespace SonarLint.VisualStudio.Integration
+{
+    internal class LanguageConverter : TypeConverter
```

Comment:
Used by JSON.NET when (de)serialising. The `Language` class is marked with the appropriate `TypeConverterAttribute`.